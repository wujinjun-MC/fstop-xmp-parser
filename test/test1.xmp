<?xpacket begin="ï»¿" id="W5M0MpCehiHzreSzNTczkc9d"?>
<x:xmpmeta xmlns:x="adobe:ns:meta/" x:xmptk="XMP Core 4.4.0-Exiv2">
 <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
  <rdf:Description rdf:about=""
    xmlns:dc="http://purl.org/dc/elements/1.1/"
    xmlns:fstop="http://www.fstopapp.com/xmp/"
   fstop:favorite="1">
   <dc:subject>
    <rdf:Bag>
     <rdf:li>foo</rdf:li>
     <rdf:li>bar</rdf:li>
     <rdf:li>baz</rdf:li>
     <rdf:li>for bar baz</rdf:li>
     <rdf:li>true</rdf:li>
     <rdf:li>false</rdf:li>
     <rdf:li>null</rdf:li>
     <rdf:li>1</rdf:li>
     <rdf:li>2</rdf:li>
     <rdf:li>3</rdf:li>
     <rdf:li>0.0</rdf:li>
     <rdf:li>wujinjun-MC</rdf:li>
    </rdf:Bag>
   </dc:subject>
  </rdf:Description>
 </rdf:RDF>
</x:xmpmeta>
<?xpacket end="w"?>
