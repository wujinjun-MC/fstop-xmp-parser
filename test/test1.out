{"test/test1.xmp":{"tags":["foo","bar","baz","for bar baz","true","false","null","1","2","3","0.0","wujinjun-MC"]}}
